nfcore_custom.config

```diff
@@ -23,9 +23,17 @@ profiles {
   phoenix      { includeConfig "${params.custom_config_base}/conf/phoenix.config" }
   shh          { includeConfig "${params.custom_config_base}/conf/shh.config" }
   uct_hex      { includeConfig "${params.custom_config_base}/conf/uct_hex.config" }
-  uppmax_devel { includeConfig "${params.custom_config_base}/conf/uppmax.config"
-                 includeConfig "${params.custom_config_base}/conf/uppmax-devel.config"
-  }
+  uppmax_devel { includeConfig "${params.custom_config_base}/conf/uppmax.config"; includeConfig "${params.custom_config_base}/conf/uppmax-devel.config" }
   uppmax       { includeConfig "${params.custom_config_base}/conf/uppmax.config" }
   uzh          { includeConfig "${params.custom_config_base}/conf/uzh.config" }
 }
+
+// If user hostnames contain one of these substring and they are
+// not running the associated profile, it will trigger a warning message
+// Should be defined here for all profiles (not within profile config)
+params {
+  // This is a groovy map, not a nextflow parameter set
+  hostnames = [
```

Comment:
Think Ive figured out whats going on by looking at the PR in `nf-core/tools`.

We would all have to explicitly specify a `hostname` value here right e.g.  
```nextflow
params {
￼	  // This is a groovy map, not a nextflow parameter set
￼	  hostnames = [
              uppmax: ['.uppmax.uu.se'],
              crick: ['.thecrick.org']
￼	  ]
￼	}


```

Reply:
Everything else looks good to me.

Reply:
Yes - that is correct :) So more PRs needed to add all of those, but seems to work nicely from my initial testing. Also needs PRs to every pipeline to add the function that uses this. Now it's being added to the template it should slowly propagate out though..

Reply:
This is what I get with test data on the dev rnaseq pipeline now:
![image](https://user-images.githubusercontent.com/465550/54298952-1d573980-45ba-11e9-9fb1-afda89388273.png)